prow/github/client.go

```diff
@@ -973,15 +973,55 @@ func (c *Client) AddLabel(org, repo string, number int, label string) error {
 	return err
 }
 
+// Use LabelNotFound to indicate that a label is not attached to an issue.
+// For example, removing a label from an issue, when the issue does not have that label.
+type LabelNotFound struct {
+	Owner, Repo string
+	Number      int
+	Label       string
+}
+
+func (e *LabelNotFound) Error() string {
+	return fmt.Sprintf("label %q does not exist on %s/%s/%d", e.Label, e.Owner, e.Repo, e.Number)
+}
+
+type githubError struct {
```

Comment:
Either inline or move the use in `func (c *Client) Merge` to use this too

Reply:
Thanks for the review @stevekuznetsov , I've added the use of the githubError struct to the Merge function.